src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const length = this.length;
+    const defaultCompare = (a, b) => {
+      if (String(a) > String(b)) {
+        return 1;
+      }
+
+      if (String(a) < String(b)) {
+        return -1;
+      }
+
+      return 0;
+    };
+    const compare =
```

Comment:
The assignment of the `compare` variable is split across two lines, which can reduce readability. Consider combining these lines for clarity.